app/reflexes/products_reflex.rb

```diff
@@ -136,9 +136,8 @@ def render_products_form_with_flash
     )
     morph :nothing
 
-    # dunno why this doesn't work.
-    # morph "#products-form", render(partial: "admin/products_v3/table",
-    #                locals: { products: products })
+    # dunno why this doesn't work. The HTML stops after the first `<col>` element, wtf?!
+    # morph "#products-form", render(partial: "admin/products_v3/table", locals:)
```

Comment:
Do you know if this question is about morph vs replace or is it about cable_ready vs stimulus_reflex?

Reply:
I don't know. When I wrote the comment I assume `morph` and `replace` should do the same thing, but I realise now those words mean different things. Time to read more of the manual...